pkg/controllers/dynakube/controller_test.go

```diff
@@ -352,14 +356,15 @@ func TestReconcileComponents(t *testing.T) {
 			injectionReconcilerBuilder:  createInjectionReconcilerBuilder(mockInjectionReconciler),
 			oneAgentReconcilerBuilder:   createOneAgentReconcilerBuilder(mockOneAgentReconciler),
 			logModuleReconcilerBuilder:  createLogModuleReconcilerBuilder(mockLogModuleReconciler),
+			extensionReconcilerBuilder:  createExtensionReconcilerBuilder(mockExtensionReconciler),
 		}
 		mockedDtc := dtclientmock.NewClient(t)
 
 		err := controller.reconcileComponents(ctx, mockedDtc, nil, dk)
 
 		require.Error(t, err)
 		// goerrors.Join concats errors with \n
-		assert.Len(t, strings.Split(err.Error(), "\n"), 4) // ActiveGate, OneAgent LogModule, and Injection reconcilers
+		assert.Len(t, strings.Split(err.Error(), "\n"), 5) // ActiveGate, Extension, OneAgent LogModule, and Injection reconcilers
```

Comment:
should we test all 9 builders in this way?

Reply:
I guess 5 is okay as we explicitly test `reconcileComponents()` here and there we call exactly the 5 reconcilers. Correct me if I am wrong.

Reply:
Or do you mean do add a new testcase where we test all 9?

Reply:
I missed the Components part in reconcileComponents. So everything is OK, thx!